final_task/rss_reader/ClassNews.py

```diff
@@ -0,0 +1,65 @@
+import re
+import html2text
+
+links_template = '\"((http|https)://(\w|.)+?)\"'
+
+
+def xml_arguments_for_class(xml_string, limit):
+    """This function receive the xml and limit of articles and returns list of dictionaries"""
+    dict_article_list = []
+    for counter, neighbor in enumerate(xml_string.iter('item')):
+        parser_dictionary = {}
+        text = html2text.HTML2Text()
+        text.ignore_images = True
+        text.ignore_links = True
+        text.ignore_emphasis = True
```

Comment:
It is better to move this outside the cycle. The way it is now. `HTML2Text` instance will be created anew every iteration